lib/exrabbit/util.ex

```diff
@@ -87,6 +87,9 @@ defmodule Exrabbit.Util do
   end
 
   defp get_formatter_module(atom) do
-    Module.concat([Exrabbit.Formatter, Atom.to_string(atom) |> String.upcase()])
```

Comment:
What is the purpose of this change? Formatter modules are supposed to follow a naming convention. I don't see a benefit in letting the users pass any module. If you already happen to have a module that implements `Exrabbit.Formatter` behaviour, it won't be a problem to write a basic wrapper module for it to stick with the convention.